### DISCUSSION TITLE
Display a note's sentence, word, and character count, unique word count, estimated time to read/speak/handwrite, and estimated readability of text

### DISCUSSION THREAD

**Discussion by sigaloid** [TriliumNext/Trilium#2188] [Show and tell]
Mostly just an upgrade to the existing one in the wiki, have just added the time to read/speak/handwrite  (~275, ~180 wpm, 68cpm). + unique word count
https://gist.github.com/sigaloid/ed8bac1254fabcc89d2eb1c5d595f539
Might update with more features (like those that exist within https://wordcounter.net/)
Was thinking about porting Dale-Chall readability formula to JS to display the reading difficulty of a note but that's for another day, today I've got a test to study for :sweat_smile: 

![image](https://user-images.githubusercontent.com/69441971/135021901-92fb6f7e-1936-495e-bcd1-a81361f14952.png)

---

If you want the Dale-Chall readability formula version, here you go: https://gist.github.com/sigaloid/c293caa1b7b5ae2cb2fd1371be733eb9

**Comment by glen-w** [comment]
Wonderful! Thanks for sharing - have implemented in my own instance :)

Would absolutely love inclusion of Dale-Chall readability formula, so hopefully your test went well!

**Reply by sigaloid** [reply]
I don't think I did well :0... But here you go! 

https://gist.github.com/sigaloid/c293caa1b7b5ae2cb2fd1371be733eb9 

I won't update the one in the header because the word list is 3000 words, so maybe a bit too big unless you specifically want it. But if you do, use this one instead.

Here's what it looks like: 
![image](https://user-images.githubusercontent.com/69441971/141152697-65acffa5-b7a8-4e61-8c42-ba685f9e7637.png)
